packages/agreement-process/src/services/agreementService.ts

```diff
@@ -437,25 +437,9 @@ export async function createAgreementLogic(
     verifiedAttributes: [],
     certifiedAttributes: [],
     declaredAttributes: [],
-    suspendedByConsumer: undefined,
```

Comment:
This change in `createAgreementLogic` is unrelated to the PR description, just a cleanup of some properties set to `undefined` (maybe there was a reason behind these, that I missed)

Reply:
I tested it and it seems to work, but maybe there was a reason or edge case that I am not aware of that required these properties to be set to `undefined` here (cc also @MalpenZibo that may have some more info)

Reply:
Still, being this unrelated, we could also just leave it as it was

Reply:
From my point of view, we could remove these properties without problem